recaf-assembler/src/main/java/me/coley/recaf/assemble/ast/arch/AbstractMemberDefinition.java

```diff
@@ -0,0 +1,46 @@
+package me.coley.recaf.assemble.ast.arch;
+
+import me.coley.recaf.assemble.ast.BaseElement;
+import me.coley.recaf.assemble.ast.meta.Signature;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class AbstractMemberDefinition extends BaseElement implements MemberDefinition{
+
+    Modifiers modifiers;
```

Comment:
Modifiers?

Reply:
all members have modifiers

Reply:
He meant that this sort of stuff can be private/final/etc in different cases. Not the `Modifiers` class literally.